### DISCUSSION TITLE
Start compare at position > 0

### DISCUSSION THREAD

**Discussion by mschwehl** [WinMerge/winmerge#1402] [Ideas]
it would be great to make winmerge to compare not at position 0 but e.g. position > 0

FILE A            FILE B
2020 xxx        2021 xxx
2020 yyy        2021 zzz
2020 uuu       2021 uuu

by start compare at position 5 you find the second line as different. 
More general, starting at second token will also find the second line

**Comment by sdottaka** [comment]
The IgnoreColumns prediffer plugin can be used to ignore specified column ranges for comparison, as shown below.

[IgnoreColumns prediffer plugin]

![image](https://user-images.githubusercontent.com/98126/178614027-6db7c3f7-7db3-457c-90db-8ec22af10465.png)
![image](https://user-images.githubusercontent.com/98126/178613608-536dc34a-fb1a-4410-81af-5702f25cb5bf.png)
![image](https://user-images.githubusercontent.com/98126/178613900-f20b900d-7a65-4550-9906-c6ce4ceecb47.png)
![image](https://user-images.githubusercontent.com/98126/178613722-6a5afd77-bf2d-412c-b7f5-5e4d01aa6556.png)
![image](https://user-images.githubusercontent.com/98126/178614115-51c1e38c-97c0-4e39-85bf-8b3cbca6704b.png)

As shown below, SelectColumns unpacker plugin can be used to extract and compare only a range of selected columns without affecting the original files.

[SelectColumns unpacked plugin]

![image](https://user-images.githubusercontent.com/98126/178612670-0a0cb34a-3a73-4f79-9fb8-c6cb06512c21.png)
![image](https://user-images.githubusercontent.com/98126/178612779-76f78fd6-74c4-46e4-89ca-8ad8d64033fc.png)
![image](https://user-images.githubusercontent.com/98126/178612813-30a37440-47f4-4cb7-a4f2-c737a3da4d1d.png)
